### ISSUE TITLE
downloader hierarchy

### ISSUE THREAD

**Issue by testlnord** [tehnar/Event-extraction#11]
Непоняная каша из downloader'ов. 
Давайте их всех засунем в отдельный модуль внутри модуля data_mining и сделаем им родителя, который будет при наследовании от него сразу помещать потомка в список downloaders. Больше родитель ничего делать уметь не будет, т.е. у него будет абстрактный метод get_articles.

см:
NotImplementedError
metaclass

**Comment by tehnar** [comment]
А зачем еще один модуль? В смысле, что сейчас в модуле data_mining только downloader-ы и есть.

**Comment by tehnar** [comment]
@testlnord 
Возникает проблема в том, что надо как-то импортировать все модули, чтобы сами классы таки создались.
Правда ли, что в любом случае придется либо прописывать все имена модулей (например, в `__all__` в `__init__.py`), либо же руками обходить все .py файлы в папке и импортировать их?

**Comment by testlnord** [comment]
Да, импортировать все равно придется. Можно, конечно, написать код, который соберет все *.py файлы из указанного каталога. Но мне это кажется странным решением, нужно будет их как-то валидировать.
